File: VB/WindowsFormsApplication741/MyFlyoutPanelToolForm.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.Utils
Imports DevExpress.Utils.Win

Namespace WindowsFormsApplication741
    Public Class MyFlyoutPanelToolForm
        Inherits FlyoutPanelToolForm

        Public Sub New(ByVal owner As Control, ByVal flyoutPanel As FlyoutPanel, ByVal options As FlyoutPanelOptions)
            MyBase.New(owner, flyoutPanel, options)
        End Sub
        Protected Overrides Function CreateHandler() As BasePopupToolWindowHandler
            Return New MyFlyoutPanelToolFormHandler(Me)
        End Function
    End Class
End Namespace

File: CS/WindowsFormsApplication741/MyFlyoutPanelToolForm.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.Utils;
using DevExpress.Utils.Win;

namespace WindowsFormsApplication741 {
    public class MyFlyoutPanelToolForm : FlyoutPanelToolForm {
        public MyFlyoutPanelToolForm(Control owner, FlyoutPanel flyoutPanel, FlyoutPanelOptions options)
            : base(owner, flyoutPanel, options) {
        }
        protected override PopupToolWindowAnimationProviderBase CreateAnimationProvider()
        {
            if (Anchor.HasFlag(AnchorStyles.Left))
            {
                return new MyPopupToolWindowLeftRightSlideAnimationProvider(this);
            }
            return base.CreateAnimationProvider();
        }
    }
}
